chatbot-core/backend/app/utils/api_response.py

```diff
@@ -13,6 +14,10 @@ def __init__(self) -> None:
 
         self.resp = resp
 
+    def set_status_code(self, status_code: int) -> "BackendAPIResponse":
+        self.resp.status_code = status_code
+        return self
+
```

Comment:
Just remove, don't need it.

Reply:
Nooooooooooooooooooooo
Remove causes error

Reply:
![image](https://github.com/user-attachments/assets/fde492a1-2e01-412d-af8a-d0c115f0793c)

Reply:
Ok I know this command. Just simply delete method `set_status_code` at each place where the class `BackendAPIResponse` appears.

Reply:
Ok https://github.com/TinhHoaSolutions-EzHR/chatbot/pull/49/commits/be1eb40bea1aa10eef0b8fe8a23169b7192ba88c